src/Microsoft.ServiceFabric.Services.Remoting/Description/MethodDescription.cs

```diff
@@ -61,10 +61,11 @@ public MethodInfo MethodInfo
      
         internal static MethodDescription Create(string remotedInterfaceKindName, MethodInfo methodInfo)
         {
-            var argumentList = new List<MethodArgumentDescription>();
+            var parameters = methodInfo.GetParameters();
+            var argumentList = new List<MethodArgumentDescription>(parameters.Length);
             var hasCancellationToken = false;
 
-            foreach (var param in methodInfo.GetParameters())
+            foreach (var param in parameters)
```

Comment:
Sure. We can initialize argumentList size to parameters.Length to reduce list resize operations because we know the argumentList.Length will either be parameters.Length (if we have no CancellationToken parameter) or parameters.Length-1 (if we have a CancellationToken parameter). Otherwise, the underlying array in the list will resize from 0->4->8->16->etc. while the list is being added to.